integration/kubernetes/untrusted_workloads/job-template.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: process-item-$ITEM
+  annotations:
+    io.kubernetes.cri-o.TrustedSandbox: "false"
+    io.kubernetes.cri.untrusted-workload: "true"
+  labels:
+    jobgroup: jobtest
+spec:
```

Comment:
`runtimeClassName: kata` ?

Reply:
@devimc that is not needed for untrusted_workloads